migrations/views.py

```diff
@@ -0,0 +1,39 @@
+from flask import Blueprint, jsonify, request
+import click
+
+from migrations.migration import Migration
+
+migration_blueprint = Blueprint('migration', __name__, cli_group=None)
+migration_manager = Migration()
+
+
+@migration_blueprint.cli.command("migrate")
+@click.argument('migration_name')
+def migrate(migration_name):
+    # Generate a new migration script
+    migration_manager.generate_migration_script(migration_name)
+
+
+@migration_blueprint.cli.command("insert")
+@click.argument('migration_name')
+def migrate(migration_name):
+    # Generate a new migration script
+    migration_manager.generate_migration_script(migration_name)
+
+
+@migration_blueprint.cli.command("upgrade")
```

Comment:
please add help strings to all of these cli.commands (and arguments), so users can review that these functions do from the command line (via `<command_name> --help`)